docs/howto/configurations-ref.rst

```diff
@@ -0,0 +1,1141 @@
+See:
+http://docs.celeryproject.org/en/master/userguide/configuration.html#std:setting-broker_transport_options
+
+
+.. _config-ref/dask:
+
+[dask]
+^^^^^^
+
+This section only applies if you are using the DaskExecutor in
+[core] section above
+
+cluster_address
+***************
+The IP address and port of the Dask cluster's scheduler.
+tls_ca
+******
+TLS/ SSL settings to access a secured Dask scheduler.
+
+tls_key
+*******
+tls_cert
+********
+
+
+
+.. _config-ref/scheduler:
+
+[scheduler]
+^^^^^^^^^^^
+
+job_heartbeat_sec
+*****************
+
+Task instances listen for external kill signal (when you clear tasks
+from the CLI or the UI), this defines the frequency at which they should
+listen (in seconds).
+
+scheduler_heartbeat_sec
+***********************
+The scheduler constantly tries to trigger new tasks (look at the scheduler section in the docs for more information). This defines how often the scheduler should run (in seconds).
+
+num_runs
+********
+The number of times to try to schedule each DAG file
+-1 indicates unlimited number
+
+
+processor_poll_interval
+***********************
+The number of seconds to wait between consecutive DAG file processing
+
+min_file_process_interval
+*************************
+after how much time (seconds) a new DAGs should be picked up from the filesystem
+
+dag_dir_list_interval
+*********************
+How often (in seconds) to scan the DAGs directory for new files. Default to 5 minutes.
+
+print_stats_interval
+********************
+How often should stats be printed to the logs
+
+scheduler_health_check_threshold
+********************************
+If the last scheduler heartbeat happened more than scheduler_health_check_threshold ago (in seconds), scheduler is considered unhealthy. This is used by the health check in the "/health" endpoint
+
+child_process_log_directory
+***************************
+
+
+scheduler_zombie_task_threshold
+*******************************
+Local task jobs periodically heartbeat to the DB. If the job has not heartbeat in this many seconds, the scheduler will mark the associated task instance as failed and will re-schedule the task.
+
+catchup_by_default
+******************
+max_tis_per_query
+*****************
+Turn off scheduler catchup by setting this to False. Default behavior is unchanged and Command Line Backfills still work, but the scheduler will not do scheduler catchup if this is False, however it can be set on a per DAG basis in the DAG definition (catchup)
+
+This changes the batch size of queries in the scheduling main loop. If this is too high, SQL query performance may be impacted by one or more of the following:
+- reversion to full table scan
+- complexity of query predicate
+- excessive locking
+
+Additionally, you may hit the maximum allowable query length for your db.
+
+Set this to 0 for no limit (not advised)
+
+
+.. _config-ref/statsd:
+
+[statsd] (https://github.com/etsy/statsd) integration settings
```

Comment:
```suggestion
``[statsd]``
```
I would prefer to keep headers simple so that it is easier to navigate the entire document.  We can add the same link in the section description. I will try to add more crosslinks in the next PR when this PR is accepted.